src/index.html

```diff
@@ -11,7 +11,176 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <div>
+          <label class="form_field">
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div>
+          <label class="form_field">
+            Name:
+            <input
+              type="text"
+              name="name"
+              maxlength="20"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div>
+          <label class="form_field">
+            How old are You?
+            <input
+              type="number"
+              name="number"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+        <div>
+          <label class="form_field">
+            Full date of birth:
+            <input
+              type="date"
+              name="birthday"
+              required
+            >
+          </label>
+        </div>
+        <div>
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="terms"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="field">
+        <legend>Registration</legend>
+        <div>
+          <label class="form_field">
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div>
+          <label>
```

Comment:
Add the 'form_field' class to the label element for consistency with other labels.